ui/pages/token-allowance/token-allowance.test.js

```diff
@@ -79,6 +79,10 @@ const state = {
   confirmTransaction: {
     txData: {},
   },
+  send: {
+    currentTransactionUUID: null,
+    draftTransactions: {},
+  },
```

Comment:
Hey not sure why should tokenAllowance need send in state.

Reply:
<ApproveContentCard/> in token-allowance is using <ConfirmGasdisplay/>, and since we are using the same component for Send and confirmation, we need to call `useSelector(getCurrentDraftTransaction)` which is breaking the token-allowance test.